_data/success-stories.yml

```diff
@@ -0,0 +1,80 @@
+cases:
+  - name: Netflix
+    link: https://medium.com/netflix-techblog/global-continuous-delivery-with-spinnaker-2a6896c23ba7
+    logo: cases-logos/netflix.jpg
+    text: “The community uptake has been tremendous! Having a platform that out-of-the-box accommodates multiple cloud providers and an extendable model to participate within, means that we are one large community rather than a bunch of micro-communities focused on various forks. This has the benefit that everyone within the community can take advantage of various innovations.”
+
+  - name: Waze
+    link: https://cloud.google.com/blog/products/gcp/guest-post-multi-cloud-continuous-delivery-using-spinnaker-at-waze
+    logo: cases-logos/waze.jpg
+    text: “Thanks to Spinnaker, developers can focus on developing business logic, rather than becoming experts on each cloud platform. Teams can track the lifecycle of a deployment using several notification mechanisms including email, Slack and SMS, allowing them to coordinate handoffs between developer and QA teams”.
+
+  - name: Avast
+    link: Avast_v3-10-15-2019.pdf
+    logo: cases-logos/avast.jpg
+    text: “Spinnaker helped the team cut down the time and work needed for the versioning of ConfigMaps and secrets. That’s a pain to do other ways. Maybe it seems simple, but it’s a big deal for our team. The whole goal here is to not wake people up in the middle of the night. The rollback and no-downtime deployment features are things everybody wants to do. "The thing that Spinnaker promises is automated canary analysis. That to me is the holy grail to enabling continuous deployment.”
+
+  - name: MarkerBot
+    link: Case%20Study%20Makerbot%20-%20Letter%20size_10-12-19.pdf
+    logo: cases-logos/markerbot.jpg
+    text: “We originally envisioned transitioning toward Kubernetes on Google Cloud over three quarters. With Spinnaker’s open source flexibility, we completed the full migration in just 2.5 months!". 
+
+  - name: Lookout
+    link: Lookout_10072019.pdf
+    logo: cases-logos/lookout.jpg
+    text: “We knew our product had game-changing potential in the government sector, but our existing deployment framework was too human-intensive and time-consuming for FedRAMP compliance. Spinnaker provided the platform, support, and delivery pipeline we needed to bring our patch deployment times up to speed.”	
+
+
+
+testimonials:
+  - name: TransUnion
+    logo: testimonials-logos/transunion.png
+    questions:
+      - question: How did you discover Spinnaker?
+        answer: Spinnaker is a multi-cloud continuous integration & deployment tool . We love the simple command line interface for administration, integration with multiple platforms and easily configurable using pipelines
```

Comment:
I think, as long as we're editing respondents' answers, we can remove sentences like this first one. "Spinnaker is a..." It's already defined elsewhere.

Reply:
+1 to removing unnecessary sentences.